### ISSUE TITLE
Browser plugin does not trigger autofill via keyboard shortcut

### ISSUE THREAD

**Issue by mvasquez** [bitwarden/clients#10687]
### Steps To Reproduce

1. Go to any login form, and focus the cursor within either of the usual input boxes (username, password)
2. Activate the keyboard shortcut defined in the browser extension to autofill.
3. Nothing happens, and it forces you to use the mouse in order to select one of the identity cards from the overlaid popover dropdown. Also, you no longer can use the keyboard via arrow keys in order to select the correct identity.

### Expected Result

Allow the user to use the keyboard via distinct mechanisms in order to authenticate efficiently or for accessibility reasons.

Allow the autofill functionality to be called on-demand or headless.

Don't keep popping up everywhere on the screen when not needed.

### Actual Result

It effectively nullifies the use of a keyboard in order to select a login card.

Also, the new popover is way more intrusive when filling out forms, it re-appears on every input you're filling in with the option of "Create identity", even though you don't intent to use that. In input intensive forms, such as surveys, registration forms, or even line-of-business applications, you don't intend to ever use/reuse identities once authenticated. However, you need to quick visibility on the next fields (now obstructed) to navigate and fill as fast as possible with the keyboard.

### Screenshots or Videos

Attached:
![01](https://github.com/user-attachments/assets/c1fba4ed-7f6a-4e5d-b003-82eeb7cea4e2)
![02](https://github.com/user-attachments/assets/0ad5c70e-2bfc-44ba-9138-42273a297f2c)

### Additional Context

I noticed this after installing version 2024.8.0.

I am a keyboard-oriented person, my typical use case is:
1. I navigate to a site
2. I try to login to said site.
3. Navigate/focus to the first field in the authentication form (usually login textbox)
4. I press a preset keyboard shortcut in order to autofill the form (when there are more than one choice for a given site, I pick the right one and hit Enter), this will typically submit the form. If the form is designed weirdly, I navigate away onto the submit button and click it either by pressing Enter or the spacebar.

All of this, without ever touching the mouse. 

### Operating System

Windows

### Operating System Version

Windows 10 22H2 19045.4780

### Web Browser

Microsoft Edge

### Browser Version

v128.0.2739.42

### Build Version

v2024.8.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by cagonzalezcs** [comment]
@mvasquez 

For the keyboard shortcut, can you please check your extension keyboard settings by visiting `edge://extensions/shortcuts`. We've seen a very persistent and difficult to resolve bug that triggers a reset/unset of the keyboard shortcut when the extension has been updated. I've done some digging into that issue, with little luck on finding a proper resolution for the problem. 

That said, the issue you're experiencing might be related to that bug. You should be able to reset the keyboard short from the referenced url above.

As for the inline menu's behavior, this is working as intended. I'll take into consideration your feedback however, but for now if this is not a desirable behavior, I'd recommend turning off the inline menu by going to `Settings > Autofill > Show autofill menu on form fields` and turning this to an `Off` state.

**Comment by mvasquez** [comment]
@cagonzalezcs thank you for your response, 

Re: keyboard shortcuts. You are right about the resetting, after updating I found that the shortcut was no longer there, and then adding to the confusion, was the fact that I set it back but in the wrong setting "Autofill the **last used identity**..." instead of using the correct one which is "Autofill the **last used login**...". So, that is no longer an issue.

Re: the inline menu's behavior, this suddenly showed up and I didn't even recall that there was a setting for turning that off and assumed it was new behavior (my bad). It's very likely that I've had it off for the longest time, and after an upgrade that setting was set back to `When field is selected (on focus)` and that threw me off big time. When you pointed me back to that setting, sure enough it wasn't in the right setting, so it's probable this setting was reset after upgrading. After turning it off, the behavior was right back to what I was expecting. This also automatically fixes the inline prompts for identity in longer forms.

Thanks again for your help, Cesar.
